hack/build.sh

```diff
@@ -50,7 +50,11 @@ function squash {
 dirs=${VERSION:-$VERSIONS}
 
 for dir in ${dirs}; do
-  IMAGE_NAME="${BASE_IMAGE_NAME}-${dir//./}-${OS}"
+  if [ "$dir" == "2.6" ]; then
```

Comment:
@jhadvig if it's the SCL conventions i don't have a huge problem with it.  if we need to go back and change the existing ones it's more of a problem (need to not break people) but we can definitely use the new scheme going forward, at a minimum.

that said, sure my preference would be to stick with the scheme we have, so I guess I would ask @hhorak if this is absolutely necessary?